core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -2154,6 +2154,24 @@ class SnappySession(_sc: SparkContext) extends SparkSession(_sc) {
     }
     (scalaTypeVal, SnappySession.getDataType(storeType, storePrecision, storeScale))
   }
+
+  /*
+     Method to add/update Structured Streaming UI Tab if cluster is running in embedded mode or
+     smart connector mode using SnappyData's Spark distribution
+  */
+  def updateStructuredStreamingUITab: Unit = {
+    try {
+      val updateUIMethod: Method = super.getClass.getMethod("updateUIWithStructuredStreamingTab")
+      updateUIMethod.invoke(this)
+    } catch {
+      case e: NoSuchMethodException =>
+        logWarning("Unable to add Structured Streaming UI Tab because" +
+            " updateUIWithStructuredStreamingTab method is not found in SparkSession class.")
+    }
+  }
+  // Call to update Structured Streaming UI Tab
+  updateStructuredStreamingUITab;
```

Comment:
Remove the semicolon. Also add empty paranthesis in the function call and function definition

Reply:
Done.